osu.Game.Rulesets.Osu/UI/OsuTouchInputMapper.cs

```diff
@@ -97,26 +97,32 @@ private void updatePositionTracking(TrackedTouch newTouch)
                 return;
             }
 
-            // ..or if the current position tracking touch was not a direct touch (this one is debatable and may be change in the future, but it's the simplest way to handle)
-            if (!positionTrackingTouch.DirectTouch)
+            // ..or if the current position tracking touch was not a direct touch (and didn't travel across the screen too far).
+            if (!positionTrackingTouch.DirectTouch && positionTrackingTouch.DistanceTravelled < 200)
```

Comment:
I'd like the `200` to be lifted to a `const` field, or some other prominent place with an explanation.

Also, in what space is `200` calculated in? If it's screen pixels, that will lead to inconsistent UX (eg. compare a 21" 4k display to a 1080p one). I think it would make the most sense if the value is calculated relative to the playfield size, taking into account screen scaling, etc.

Reply:
It's already gameplay pixels, yep.